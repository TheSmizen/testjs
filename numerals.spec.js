const { numerals, numbers } = require("./numerals"); //.js is implied, { myThing } says import only this thing //"object destructuring"
describe("Testing number => numerals ", () => {
  it("Should turn numbers into numerals e.g. 2000 => MM", () => {
    expect(numerals(1999)).toBe("MCMXCIX");
    expect(numerals(2000)).toBe("MM");
    expect(numerals(2)).toBe("II");
    expect(numerals(15)).toBe("XV");
    expect(numerals(75)).toBe("LXXV");
  });
});
describe("Testing numerals => numbers ", () => {
  it("Should turn numerals into numbers e.g. MM => 2000", () => {
    expect(numbers("MCMXCIX")).toBe(1999);
    expect(numbers("MM")).toBe(2000);
    expect(numbers("II")).toBe(2);
    expect(numbers("XV")).toBe(15);
    expect(numbers("LXXV")).toBe(75);
  });
});
