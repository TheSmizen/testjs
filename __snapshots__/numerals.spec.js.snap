// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing numerals => numbers  Throws an error when an invalid string is supplied 1`] = `"Oops! You gave an invalid string."`;
